### PR TITLE
Add transpiling build process. Fixes #19

### LINKED ISSUE #19: Compatibility with screen readers on touch devices
I’ve made a demo to test the polyfill in VoiceOver on iPhone: https://output.jsbin.com/voxopus/quiet. When I swipe left/right to move to the previous/next item, it doesn’t skip over inert elements. To be honest, I’m not sure if it should. Someone with experience with screen readers on touch devices would know if swiping horizontally should skip inert elements.

### PR DESCRIPTION
**Pull request by robdodson**
Adds Babel and Uglify to the build pipeline. Updated readme to indicate which version of the polyfill folks should use.

### CODE REVIEW

**Comment by robdodson** [comment]
@alice PTAL

**Comment by robdodson** [comment]
huzzah! tests pass! 
giphy

File: test/index.html [hunk 1]
```diff
@@ -64,7 +64,7 @@
   </script>
 
   <!-- The script under test -->
-  <script src="../inert.js"></script>
+  <script src="../dist/inert.js"></script>
```
Comment (alice) [inline test/index.html]: Does this mean that presubmit tests will test against the dist version?

**Comment by alice** [comment]
Basically happy to merge except I'm confused about the tests.

File: README.md [hunk 2]
```diff
@@ -185,6 +185,9 @@ implementers may get useful functionality into the hands of developers sooner wh
 
 ## Notes on the polyfill
 
+The `dist` directory contains two versions of the polyfill. `inert.js` is the debug version, containing an inlined sourcemap, `inert.min.js` is
```
Comment (alice) [inline README.md]: Mention that these are both generated from the ES6 source, which can be used directly if desired?

File: test/index.html (hunk 1 above)
Comment (robdodson) [inline test/index.html]: Yes. I figured if that's what we're actually distributing then it's what should be tested. The npm test commandwhich Travis uses  will now generate this build before running tests

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+/*
+ * Gulp 4 lets you use `exports` module notation to declare tasks. Wheeeeee!
+ */
+exports.clean = clean;
+exports.scripts = scripts;
+exports.watch = watch;
+
+gulp.task('default', gulp.series(clean, scripts));
```
Comment (alice) [inline gulpfile.js]: Missing newline

File: test/index.html (hunk 1 above)
Comment (alice) [inline test/index.html]: We're distributing both though right?

Comment (robdodson) [inline test/index.html]: yes

Comment (alice) [inline test/index.html]: So why test this one in particular when it requires a build step?

Comment (robdodson) [inline test/index.html]: Depends on the browsers we test I guess. If we want to verify things work in IE10 we'd have to always test the built version

**Pull request merged**